bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -205,18 +206,23 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
                 };
             });
 
+            languageCode = mainLanguageCode;
+            if (host.get('request').params.languageCode) {
```

Comment:
It could be stored in a variable like `var requestParams = host.get('request').params;`. That's a small perf optimisation.